tools/dataset_converters/prepare_kitti2012.py

```diff
@@ -49,11 +49,10 @@ def main():
             img1_path=i_img1, img2_path=i_img2, flow_fw_path=i_flow_fw)
         data_list.append(data_info)
 
-    metainfo = dict(dataset='KITTI2012', subset='train', sparse=True)
-    annotation_file = osp.join(args.save_dir, 'KITTI2012_train.json')
```

Comment:
Lack of `mmcv.mkdir_or_exist(args.save_dir)`, leading to an error when specifying `--save-dir`.